python/core/auto_generated/editform/qgsattributeeditorcontainer.sip.in

```diff
@@ -112,6 +134,30 @@ show or hide this container based on an expression incorporating
 the field value controlled by editor widgets.
 
 .. versionadded:: 3.0
+%End
+
+    QgsOptionalExpression collapsedExpression() const;
```

Comment:
This should really use QgsPropertyCollection, but I can see that you've modelled it off the existing visibility expression, so you get a pass here...

(maybe we should sip_skip it though to make a future move to QgsPropertyCollection easier)

Reply:
done